core/src/main/java/psiprobe/controllers/RenderChartController.java

```diff
@@ -162,9 +162,9 @@ protected ModelAndView handleRequestInternal(HttpServletRequest request,
 
       final XYLine3DRenderer renderer = new XYLine3DRenderer();
       renderer.setDrawOutlines(true);
-      renderer.setLinesVisible(true);
-      renderer.setShapesVisible(true);
-      renderer.setStroke(new BasicStroke(2));
+      renderer.setSeriesLinesVisible(0, true);
+      renderer.setSeriesShapesVisible(0, true);
+      renderer.setSeriesStroke(0, new BasicStroke(2));
```

Comment:
Just looking at the API, it kinda looks like these should be done in a for-loop now:

```java
for (int i = 0; i < seriesMaxCount; i++) {
  renderer.setSeriesLinesVisible(i, true);
  renderer.setSeriesShapesVisible(i, true);
  renderer.setSeriesStroke(i, new BasicStroke(2));
}
```

I'd also suggest testing it locally just to be sure it still works.